sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/Utils.java

```diff
@@ -31,8 +31,8 @@
                     document,
                     obj.get(Constants.PARTITION_KEY_FIELD_NAME).getAsString(),
                     obj.get(Constants.ID_FIELD_NAME).getAsString(),
-                    obj.get(Constants.ETAG_FIELD_NAME).getAsString(),
-                    obj.get(Constants.TIMESTAMP_FIELD_NAME).getAsLong());
+                    obj.has(Constants.ETAG_FIELD_NAME) ? obj.get(Constants.ETAG_FIELD_NAME).getAsString() : "",
```

Comment:
Should we put default values in constants or is it ok to leave here?  @guperrot ?

Reply:
I don't think an empty string or a zero needs a constant (and in this case, the value is not going to be re-used or verified somewhere else). I know, it's a contentious issue akin to tabs vs spaces, squash vs non-squash and a number of other ones. But doesn't seem like it's too important in this particular case which way to go.

Reply:
I don't remember creating a constant for an empty string so I don't mind this one.